### PR TITLE
docs: Update README stating this project is in maintenance mode

### LINKED ISSUE #1064: Update README to state this project is now in maintenance mode
Now that we have released [Clear Containers 3.0](https://github.com/clearcontainers/runtime/releases/tag/3.0.0), this codebase (CC 2.x) is now deprecated and the project is in maintenance mode.

Update the `README.rst` to state this and encourage users to switch to the [new runtime](https://github.com/clearcontainers/runtime).

### PR DESCRIPTION
**Pull request by jodh-intel**
Now that we have released [Clear Containers
3.0](https://github.com/clearcontainers/runtime/releases/tag/3.0.0),
this codebase (CC 2.x) is now deprecated and the project is in
maintenance mode.

Updated the `README.rst` to state this and encourage users to switch to
the [new runtime](https://github.com/clearcontainers/runtime).

Fixes #1064.

Signed-off-by: James O. D. Hunt <james.o.hunt@intel.com>

### CODE REVIEW

**Comment by jodh-intel** [comment]
ping @jcvenegas, @sameo, @grahamwhaley.

**Comment by grahamwhaley** [comment]
The words look fine - but the file is not rendering for me when viewed on github on your branch. I suspect it should, so wonder if somehow something has broken the rst'ness of the file - can you check that for me?

**Comment by grahamwhaley** [comment]
and you'll need @iphutch as well.

**Comment by jcvenegas** [comment]
@jodh-intel hey james,  looks good :) , but also I could not seen the text well rendered, even taking a look at your branch directly in your fork. I wonder if the `warning` directive is not recognized by github render.

**Comment by jodh-intel** [comment]
Branch updated.

Hi @iphutch - can you review please?

**Comment by grahamwhaley** [comment]
yay, that renders for me now.
lgtm

**Comment by jodh-intel** [comment]
Sorry - I had a better idea - branch updated again ;)

File: README.rst [hunk 1]
```diff
@@ -1,448 +1,23 @@
...
-.. [#oci-vm-config-issue]
-   https://github.com/opencontainers/runtime-spec/pull/405
+All users are encouraged to switch to the new ``cc-runtime`` runtime.
+Installation and upgrade information is available on `the new runtime's wiki
+<https://github.com/clearcontainers/runtime/wiki>`_.
 
-.. [#oci-runtime-cli-clarification]
-   https://github.com/opencontainers/runtime-spec/issues/434
+To see the original project ``README``, click
+`here <https://github.com/01org/cc-oci-runtime/blob/master/README-original.rst>`_.
```
Comment (grahamwhaley) [inline README.rst]: nit: you should be able to make this a relative reference like [this](README-original.rst) (or the rst equiv?) I would think? Means it is maybe more future proof (and would de-ref correctly in your pending PR ;-)

**Review by grahamwhaley — APPROVED** [review]
Yeah, that's a better idea.
tiny nit
lgtm

File: README.rst (hunk 1 above)
Comment (jodh-intel) [reply README.rst]: Agreed. Branch de-nitted! ;)

**Comment by jodh-intel** [comment]
@jcvenegas - could you review please?

**Comment by jodh-intel** [comment]
or @devimc ;)

**Review by jcvenegas — APPROVED** [review]

**Comment by jodh-intel** [comment]
Hi @iphutch - please can you review this so we can land it?

**Review by rcaballeromx — APPROVED** [review]

**Pull request merged**